### ISSUE TITLE
Allow us to change tenant in admin

### ISSUE THREAD

**Issue by phalt** [tomturner/django-tenants#263]
The recent introduction of this change has made it very hard for us to use the django admin properly: https://github.com/tomturner/django-tenants/pull/227

For now, we've dealt with the problem by copy/pasting the original Django admin site template. Having an option to turn this on or off, or alternatively - to pick the tenant we want to use in the admin, would be nice :)

**Comment by ImplicitCoder** [comment]
+1 for making the color change optional. I will see if I have time to implement this in the near future.
Meanwhile, I have a slightly lighter workaround than phalt:
> For now, we've dealt with the problem by copy/pasting the original Django admin site template. 

In templates/admin/appname I added a base.html with following code:
``` 
{% extends 'admin/base.html' %}
{% block extrahead %}  <!-- must be in extrahead, putting this in extrastyle will not work -->
    <style>
     .module caption.tenant-app {
        background: #79aec8;  <!-- original django admin color -->
    }
    .module caption.public-disabled {
        background: #c6c6c6;
    }
    </style>
{% endblock %}
```

**Comment by tomturner** [comment]
Fixed in #272
